nextflow_schema.json

```diff
@@ -903,6 +991,20 @@
             },
             "fa_icon": "fab fa-app-store-ios"
         },
+        "multi_module_settings": {
+            "title": "Multi module settings",
+            "type": "object",
+            "description": "These parameters affect multiple modules in the pipeline.",
+            "default": "",
+            "properties": {
+                "round_digits": {
```

Comment:
Just use the existing 'reporting_options' section and mention in the description that it just affects some tabular outputs in the report HTML.